### DISCUSSION TITLE
Prevent from output window to pop-up on save?

### DISCUSSION THREAD

**Discussion by ferarguelles** [glenn2223/vscode-live-sass-compiler#251] [Q&A] [answered]
First of all, kudos for the great work on keeping this extension alive and with some incredible enhances over the old version.

I have a question I couldn't find in the FAQs: 
Is there a way to prevent the output window in Visual Studio Code to open every time I the compilation process goes through? 

I know in the settings you can turn off the output altogether by setting Show Output Window On to 'none'. But this removes all output completely. In the old version of the extension the output was always logged in the output window, it was just that the window didn't show.

Thank you for your kind reply.

**Comment by glenn2223** [comment, marked as answer]
You can change the level at which the output pops up with the [liveSassCompile.settings.showOutputWindowOn](https://github.com/glenn2223/vscode-live-sass-compiler/blob/master/docs/settings.md#livesasscompilesettingsshowoutputwindowon) setting. You can set it to `Warning` so it only pops up on warning or error messages.

I hadn't seen that it was still logging, just not popping up. I wanted this for my version, so I'm glad to see it did this originally.

**Reply by ferarguelles** [reply]
This behaviour is exactly what I was expecting – makes total sense. Thank you so much.
